initdb.py

```diff
@@ -1,5 +1,3 @@
-# Initializes a MySQL Database
-
 import os
```

Comment:
@asclines It really doesn't like the whitespace left between the Doc-string comments?